src/components/modal/examples/components/FancyModal.vue

```diff
@@ -0,0 +1,197 @@
+          <div
+            class="content green"
+            role="document"
+          >
+            <div class="icon">
+              <cdr-tooltip position="left">
+                <template #trigger>
+                  <membership-icon />
+                </template>
+                Hey look at me, I'm an absolutely positoned and transformed icon!
+              </cdr-tooltip> 
+            </div>
+            <cdr-text
+              tag="h3"
+              class="cdr-text-dev--heading-serif-600"
+              data-ui="signup-modal-title"
+            >
+              <p class="nomargin">
+                You belong here.
+              </p>
+              <p class="nomargin">
+                And out there.
+              </p>
+            </cdr-text>
+            <p class="subheading">
+              Be a part of REI Co-op, a community of 20 million people making a different type of outdoor company. Together we're sharing, exploring and protecting the place we love. For today and every day.
+            </p>
+            <cdr-button
+              :full-width="true"
+              modifier="secondary"
+            >
+              Explore membership benefits
+            </cdr-button>
+          </div>
+        </div>
+      </template>
+    </cdr-modal>
+  </div>
+</template>
+
+<script>
+import * as Components from 'srcdir/index';
+import MembershipIcon from '../../../icon/comps/membership.vue';
+
+export default {
+  name: 'FancyModal',
+  components: {
+    ...Components,
+    MembershipIcon,
+},
+  data() {
+    return {
+      modalOpened: false,
+    };
+  },
+};
+</script>
+<style lang="scss">
+@import "node_modules/@rei/cdr-tokens/dist/scss/cdr-tokens.scss";
+.membership-acquisition-imodal {
+  // class to override cedar default class to allow background content more visible to user
+  &--overlay-lighter {
+    backdrop-filter: initial;
+    background-color: rgba(26, 26, 26, 0.6);
+  }
+  &__wrapper {
+    padding: $cdr-space-one-x;
+    .close-button-wrapper {
+      position: absolute;
+      right: 0;
+      z-index: 1;
+    }
+    .modal-close-button {
+      fill: $cdr-color-background-label-secondary-active;
+      @include cdr-sm-mq-down {
+        padding-top: $cdr-space-one-x;
+      }
+    }
+    padding: $cdr-space-one-x;
+
+    // Target modal container and avoid full screen behaviour on smaller resolutions.
+    div[role="dialog"] {
+      @include cdr-sm-mq-down {
+        border-radius: $cdr-radius-softer;
+        max-width: 315px;
+        min-height: auto;
+        position: relative;
+      }
+    }
+
+    .content-wrapper {
+      display: flex;
+      flex-direction: column;
+      @include cdr-sm-mq-up {
+        flex-direction: row;
+      }
+      .image {
+        background-image: url("https://www.rei.com/assets/membership/benefits/modal/parenting/live.jpg");
+        background-position: center top;
+        background-size: 100% auto;
+        background-position: 0% 55%;
+        height: 180px;
```

Comment:
This was basically a copy/paste from `@rei/member-accquistion-modal`, I hesitated to change too much of it because I wanted to maintain parity with the real thing

Reply:
thats fine - as I said just a nit